src/dict.c

```diff
@@ -906,7 +915,11 @@ void dictSetUnsignedIntegerVal(dictEntry *de, uint64_t val) {
 
 void dictSetDoubleVal(dictEntry *de, double val) {
     assert(entryHasValue(de));
-    de->v.d = val;
+    if (entryIsEmbedded(de)) {
```

Comment:
Seems like we need to fix this for all the `dictSet*Val` methods.

Reply:
Merge with unstable. This is handled by Ping's change https://github.com/valkey-io/valkey/pull/749/

Reply:
Ok, then i just need to reslove the conflicts.